src/effects/backends/builtin/compressoreffect.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include "effects/backends/effectprocessor.h"
+#include "engine/effects/engineeffect.h"
+#include "engine/effects/engineeffectparameter.h"
+#include "util/class.h"
+#include "util/defs.h"
+#include "util/sample.h"
+#include "util/types.h"
+
+namespace {
+constexpr CSAMPLE_GAIN kMakeUpAttackCoeff = 0.03f;
+constexpr CSAMPLE_GAIN kMakeUpTarget = -3.0f;
+
+double calculateBallistics(double paramMs, const mixxx::EngineParameters& engineParameters) {
```

Comment:
Unused?

Reply:
No, I use it on these lines: 117,119,193,201 in compressoreffect.cpp

Reply:
Ah, the issue is that it is unused in very cpp file that includes the h file. Can we move the whole namespace to the CPP file?

Reply:
Yes. sure. I've moved it to CPP file.